src/decoders/mrw.rs

```diff
@@ -46,7 +46,7 @@ impl<'a> MrwDecoder<'a> {
           }
         }
         0x545457 => { // TTW
-          // Base value for offsets needs to be at the beginning of the 
```

Comment:
Turns out most of these whitespace changes were introduced in the initial commit. When the rest is done I will squash all the commits and fix the whitespace changes.